scania-digital-design-system/sdds
# fix(breadcrumb): improvements

**Describe pull-request**  
Aligned the breadcrumb item with our sdds-link.

**Solving issue**  
Fixes: -

**How to test**  
1. Go to storybook link below
2. Check in Components -> Breadcumb -> Web Component
3. Check the prop descriptions

**Suggested test steps**
